src/components/ThumbnailsPanel/ThumbnailsPanel.js

```diff
@@ -432,6 +434,10 @@ class ThumbnailsPanel extends React.PureComponent {
             this.setState({
               height: bounds.height,
               width: bounds.width,
+            }, () => {
+              this.setState({
```

Comment:
Can we make a `handleResize` function and calculate the number of columns there, and just setState once? I don't think it's necessary to rerender the component twice. (one for setting the dimension, one for setting the columns)

Reply:
done